### PR TITLE
Chores/CS fixes frontend

### PR DESCRIPTION
**Pull request by devmount**
## Description of the Change

This change applies eslint fixes to the frontend.

### CODE REVIEW

File: frontend/src/elements/ListPagination.vue [hunk 1]
```diff
@@ -64,15 +60,15 @@ const goto = (index) => {
...
-const isVisibleInnerPage = (p) => (currentPage.value == 0 && p == 3)
-		|| ((currentPage.value == 0 || currentPage.value == 1) && p == 4)
-		|| (p > currentPage.value - 1 && p < currentPage.value + 3)
-		|| ((currentPage.value == pageCount.value - 1 || currentPage.value == pageCount.value - 2) && p == pageCount.value - 3)
-		|| (currentPage.value == pageCount.value - 1 && p == pageCount.value - 2)
-    || p == pageCount.value;
+const showPageItem = (p) => pageCount.value < 6 || p === 1 || p === 2 || isVisibleInnerPage(p) || p === pageCount.value - 1;
+const showFirstEllipsis = (p) => pageCount.value >= 6 && currentPage.value > 2 && p === 2;
+const showPageItemLink = (p) => pageCount.value < 6 || p === 1 || isVisibleInnerPage(p);
+const showLastEllipsis = (p) => pageCount.value >= 6 && currentPage.value < pageCount.value - 3 && p === pageCount.value - 1;
```
Comment (MelissaAutumn) [inline frontend/src/elements/ListPagination.vue:73]: We may need to clean this up in a post-this-pr PR.

**Review by MelissaAutumn — APPROVED** [review]
Looks good, thanks!

File: frontend/src/elements/ListPagination.vue (hunk 1 above)
Comment (devmount) [reply frontend/src/elements/ListPagination.vue:73]: Agreed. Past me is to blame here 😅 If you have any improvement suggestions, please just apply them!

Comment (MelissaAutumn) [reply frontend/src/elements/ListPagination.vue:73]: Haha, well it works for now! I'll give it some thought.

**Pull request merged**